common/beerocks/tlvf/yaml/beerocks/tlvf/beerocks_message_action.yaml

```diff
@@ -238,21 +238,18 @@ eActionOp_APMANAGER:
   ACTION_APMANAGER_CLIENT_BSS_STEER_REQUEST: 48
   ACTION_APMANAGER_CLIENT_BSS_STEER_RESPONSE: 49
   ACTION_APMANAGER_CLIENT_RX_RSSI_MEASUREMENT_CMD_RESPONSE: 50
```

Comment:
I guess this should be part of the commit which removed the 4addr connection flow (nitpick)

Reply:
I preferred to split commit that is related to tlvf to make the review more convenient.